src/main/kotlin/org/ozwillo/dcimporter/web/Routes.kt

```diff
@@ -43,6 +43,11 @@ class Routes(private val statusHandler: StatusHandler,
                     POST("/{siret}/consultation/{reference}/piece", marchePublicHandler::createPiece)
                     PUT("/{siret}/consultation/{reference}/piece/{uuid}", marchePublicHandler::updatePiece)
                     DELETE("/{siret}/consultation/{reference}/piece/{uuid}", marchePublicHandler::deletePiece)
+
+                    //Registres
+                    POST("/{siret}/registre/type/{type}/{reference}", marchePublicHandler::refreshDatacoreRegistreForGivenConsultation)
+                    GET("/{siret}/registre/type/{type}/{reference}", marchePublicHandler::getRegistreForConsultation)
```

Comment:
For now the "registre" word is keeped. We'll change these endpoints depending on marche public's opinion about it. 
For record, I changed the get detail resume endpoint to ".../marchepublic:retrait_0/resume" because I need to have something different from ".../{type}".